tests/unit/test_gce_clients.py

```diff
@@ -0,0 +1,344 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import logging
+
+import pytest
+from aioresponses import aioresponses
+
+from gordon_janitor_gcp import exceptions
+from gordon_janitor_gcp import gce_clients
+
+
+class TestPagingGoogleClientBase:
```

Comment:
This test module uses test classes in order to get some feedback about how others feel about using test classes. It's a nice way to group things together, but it makes using `pytest.mark.parametrize` somewhat more difficult to use and adds more complexity over using test functions like we've always done.

Once I get some feedback, I'll either rework this into simple test functions or see about making other tests into classes.